Composer/packages/client/src/components/BotRuntimeController/BotController.tsx

```diff
@@ -277,31 +278,32 @@ const BotController: React.FC<BotControllerProps> = ({ onHideController, isContr
           </DefaultButton>
         </DisableFeatureToolTip>
         <div ref={onboardRef} css={[iconSectionContainer, disableStartBots ? disabledStyle : '']}>
-          <IconButton
-            ariaDescription={formatMessage('Start and stop local bot runtimes')}
-            data-testid="StartBotsPanel"
-            disabled={disableStartBots}
-            iconProps={{
-              iconName: 'List',
-            }}
-            styles={{
-              root: {
-                color: NeutralColors.white,
-                height: '36px',
-                background: isControllerHidden ? CommunicationColors.tint10 : transparentBackground,
-                selectors: {
-                  ':disabled .ms-Button-icon': {
-                    opacity: 0.6,
-                    backgroundColor: CommunicationColors.tint10,
-                    color: `${NeutralColors.white}`,
+          <TooltipHost content={startStopLabel} directionalHint={DirectionalHint.bottomCenter}>
```

Comment:
@beyackle quick clarification, Do tooltips show up if the underlying button is disabled?

Reply:
I asked, and in this case they do.